geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/AbstractPartitionedRepositoryManager.java

```diff
@@ -111,6 +121,10 @@ protected IndexRepository computeRepository(Integer bucketId) {
     return repo;
   }
 
+  protected void allowRepositoryComputation() {
```

Comment:
Nitpick isn't this representing that the repository is ready for use?  Should rename the latch and method to reflect this, such as readyForUse and enableRepositoryForUse?